com.mattermost.Desktop.json

```diff
@@ -0,0 +1,68 @@
+{
+    "app-id": "com.mattermost.Desktop",
+    "base": "io.atom.electron.BaseApp",
+    "base-version": "18.08",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "18.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "mattermost-flatpak",
+    "separate-locales": false,
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--device=dri",
+        "--filesystem=home",
+        "--talk-name=org.freedesktop.Notifications"
+    ],
+    "modules": [
+        {
+            "name": "mattermost-desktop",
+            "buildsystem": "simple",
+            "build_commands": [
+                "mkdir -p /app/main /app/bin",
+                "cp -ra main/* /app/main/",
+                "install mattermost.sh /app/bin/mattermost-flatpak",
+                "install -Dm644 com.mattermost.Desktop.desktop /app/share/applications/com.mattermost.Desktop.desktop",
+                "install -Dm644 com.mattermost.Desktop.appdata.xml /app/share/appdata/com.mattermost.Desktop.appdata.xml",
+                "install -Dm644 /app/main/icon.png /app/share/icons/hicolor/512x512/apps/com.mattermost.Desktop.png"
+            ],
+            "sources" : [
+                {
+                    "type": "script",
+                    "dest-filename": "mattermost.sh",
+                    "commands": ["exec env TMPDIR=$XDG_CACHE_HOME /app/main/mattermost-desktop \"$@\""]
```

Comment:
I believe you might have inherited the `TMPDIR=$XDG_CACHE_HOME` from some other manifest – maybe you can delete it and see if it's actually used or not.

Other than that, looks fine :tada:

Reply:
Yeah, seem to work fine without this environment variable.